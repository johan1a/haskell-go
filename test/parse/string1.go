package main
import "fmt";
func main() {
"helloworld"
}
