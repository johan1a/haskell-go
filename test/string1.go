package main;
func main() {
"helloworld"
}
