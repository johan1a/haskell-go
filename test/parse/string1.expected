SourceFile (Package "main") [ImportDecl [I1 "fmt"]] [TopLevelDecl2 (FunctionDecl2 "main" (Signature1 []) (Block [SimpleStmt (ExpressionStmt (UnaryExpr (PrimaryExpr (PrimaryExpr1 (Operand1 (BasicLit (StringLit "helloworld")))))))]))]
